buildenv/jenkins/testJobTemplate

```diff
@@ -0,0 +1,164 @@
+#!groovy
+
+/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/**
+ * A template that defines a test job.
+ */
+
+if (!binding.hasVariable('JDK_IMPL')) JDK_IMPL = "openj9"
+if (!binding.hasVariable('ARTIFACTORY_SERVER')) ARTIFACTORY_SERVER = ""
+
+if (binding.hasVariable('LEVELS')) {
+	LEVELS = LEVELS.split(',')
+} else {
+	LEVELS = ['sanity','extended','special']
+}
+
+if (binding.hasVariable('JDK_VERSIONS')) {
+	JDK_VERSIONS = JDK_VERSIONS.split(',')
+} else {
+	JDK_VERSIONS = [8,11,12]
+}
+
+if (binding.hasVariable('GROUPS')) {
+	GROUPS = GROUPS.split(',')
+} else {
+	GROUPS = ['functional','system']
```

Comment:
Is purpose of GROUPS used for job naming and BUILD_LIST hint?
Fallback to just these 2 groups, functional, system, is for ci.eclipse.org/openj9 builds?

Reply:
I will add the rest as the default.
yes, ci.eclipse.org/openj9 builds only run functional and system atm.